### PR TITLE
Added basic documentation for MFA.

### PR DESCRIPTION
**Pull request by alesstimec**

### CODE REVIEW

File: docs/mfa.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+WebAuthn is currently supported in Google Chrome, Mozilla Firefox, Microsoft Edge and Apple Safari (preview) web browsers, as well as Windows 10 and Android platforms.
+
+For more info see [link](https://caniuse.com/?search=webauthn).
+
+## Supported authenticators
+
+Candid supports WebAuthn multi-factor authentications, which requires uses to register an external authenticator that supports [FIDO2](https://fidoalliance.org/fido2/fido2-web-authentication-webauthn/) such as Yubikey 5.
+
+First time a user logs in using an identity provider that is configured to require MFA, the user will be required to register an external authenticator. Following successful registration the user will be able to register multiple other authenticators.
```
Comment (mhilton) [inline docs/mfa.md]: "The first"?

File: docs/mfa.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+On subsequent logins user will be required to present one of the registered authenticators befor completing the login process.
+
+### Lost authenticators
+
+Should the user lose all registered authenticators, the Candid admin can user the **clear-mfa-credentials** command, which will de-register all user's authenticators. Next time the user will be required to register a new authenticator.
```
Comment (mhilton) [inline docs/mfa.md]: Maybe put in an example command line for this?

**Review by mhilton — APPROVED** [review]

File: docs/mfa.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+For more info see [link](https://caniuse.com/?search=webauthn).
+
+## Supported authenticators
+
+Candid supports WebAuthn multi-factor authentications, which requires uses to register an external authenticator that supports [FIDO2](https://fidoalliance.org/fido2/fido2-web-authentication-webauthn/) such as Yubikey 5.
+
+First time a user logs in using an identity provider that is configured to require MFA, the user will be required to register an external authenticator. Following successful registration the user will be able to register multiple other authenticators.
+
+On subsequent logins user will be required to present one of the registered authenticators befor completing the login process.
```
Comment (edumucelli) [inline docs/mfa.md]:
```suggestion
On subsequent logins user will be required to present one of the registered authenticators before completing the login process.
```

**Review by edumucelli — APPROVED** [review]

File: docs/mfa.md (hunk 3 above)
Comment (alesstimec) [reply docs/mfa.md]: done

File: docs/mfa.md (hunk 2 above)
Comment (alesstimec) [reply docs/mfa.md]: done

File: docs/mfa.md (hunk 1 above)
Comment (alesstimec) [reply docs/mfa.md]: done

**Pull request merged**